libraries/ts-command-line/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "forceConsistentCasingInFileNames": true,
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "types": [
+      "node",
+      "es6-collections"
+    ]
+  },
```

Comment:
Adding strict null checks is a good idea, but outside the scope of this PR (which is supposed to bring over the current code and get it building).
#WontFix